KabuKit/Classes/core/Swift/SceneCollection.swift

```diff
@@ -1,6 +1,6 @@
 import Foundation
 
-class SceneCollection<Stage> {
+public class SceneCollection<Stage> {
```

Comment:
https://github.com/crexista/KabuKit/blob/master/KabuKit/Classes/core/Swift/SceneSequence.swift#L12
`SceneSquence` is public so this must be public too.